src/main/scala/codacy/codesniffer/docsgen/VersionsHelper.scala

```diff
@@ -0,0 +1,25 @@
+package codacy.codesniffer.docsgen
+import java.io.FileInputStream
+import java.util.Properties
+
+object VersionsHelper {
+
+  private[this] val properties = {
+    val prop = new Properties()
+    prop.load(new FileInputStream(".versions.properties"))
+    prop
+  }
+
+  lazy val codesniffer = properties.getProperty("php-codesniffer")
+  lazy val magento = properties.getProperty("magento-eqp")
+  lazy val wordpress = properties.getProperty("wordpress")
+  lazy val phpCompatibility = properties.getProperty("php-compatibility")
+
+}
+
+object Test extends App {
```

Comment:
Do we need this? If yes can we get a better name?

Reply:
We don't, I just forgot to delete it 🙃